### PR TITLE
Documentation Nov21

### PR DESCRIPTION
**Pull request by mturilli**
This is a clean start with part of the structure we are discussing at #2454. In order to create the the notebooks, I would like to be able to use my editor but, as RP makes assumption about what is able to write to disk (caching or similar) I am not able to.

### CODE REVIEW

**Comment by andre-merzky** [comment]
> In order to create the the notebooks, I would like to be able to use my editor but, as RP makes assumption about what is able to write to disk (caching or similar) I am not able to.

Not sure I understand that part - what is the problem your editor is having with RP?  That it write log files etc. while running the notebook?

File: docs/source/quickstart.ipynb [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  {
+   "cell_type": "markdown",
+   "id": "8f84cf1f",
+   "metadata": {},
+   "source": [
+    "### x.1.1. `venv`"
+   ]
+  },
+  {
+   "cell_type": "raw",
```
Comment (mtitov) [inline docs/source/quickstart.ipynb:38]: "raw" cell type is not shown in GitHub viewer... (if is opened within GitHub these cells will look empty)

File: docs/source/quickstart.ipynb [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+   "source": [
+    "RP uses **MongoDB** as one of the communication layers to store and retrieve operational data during the execution of an application. The MongoDB server must be reachable from **both**, the host that runs the RP application and the target resource which runs the pilots. \n",
+    "\n",
+    "MongoDB connection string is of the following format(s):\n",
+    "- not-secure connection: `mongodb://user:pass@host[:port]/[dbname]`\n",
+    "- secure connection: `mongodb+ssl://user:pass@host[:port]/[dbname]?tlsAllowInvalidCertificates=True`\n",
+    "\n",
+    "RP reads the connection string from a corresponding environment variable:\n",
+    "```bash\n",
+    "export RADICAL_PILOT_DBURL=\"<connection_string>\"\n",
```
Comment (mtitov) [inline docs/source/quickstart.ipynb:132]: add an example of the tunneling command?

**Comment by andre-merzky** [comment]
@vrpascuzzi : can you please try to build the documentation in this branch?  Thanks!

**Comment by andre-merzky** [comment]
@vrpascuzzi : sorry for the noise - please ignore above request...

**Comment by andre-merzky** [comment]
- first review of nb3 needs to be completed
  - then apply similar changes to other sections

**Comment by andre-merzky** [comment]
Ok for me IIFF you are sure that there is nothing to be salvaged in this PR.  There are many changes.  @mturilli, @mtitov, @AymenFJA: the changes are from your commits, please comment.

**Comment by kartikmodi** [comment]
@AymenFJA @mtitov @mturilli @andre-merzky All the commits from the past are present in `docs/nb_section3` branch. Discarding of this branch should make no difference, as we'll merge `docs/nb_section3` branch in devel directly in future.

Lastly, docs/nov21 is not a very documentation build either. There's lot of dangling notebooks/configs which are not really complete to call it a `mergeable`.

**Comment by andre-merzky** [comment]
> @AymenFJA @mtitov @mturilli @andre-merzky All the commits from the past are present in `docs/nb_section3` branch. Discarding of this branch should make no difference, as we'll merge `docs/nb_section3` branch in devel directly in future.
> 
> Lastly, docs/nov21 is not a very documentation build either. There's lot of dangling notebooks/configs which are not really complete to call it a `mergeable`.

I think what @kartikmodi tries to suggest is that this PR be closed in favor of #2559 which would then be redirected t `devel`.  I am ok with that approach.  @mtitov, @mturilli ?

**Pull request closed without merging**